pytorch_blade/src/compiler/mlir/converters/torch_mlir_op_filter.cpp

```diff
@@ -113,24 +113,26 @@ const std::unordered_set<std::string> &GetTorchMlirWhiteList() {
       "torch_blade::fake_quant"
     };
 
+  auto custom_ops = env::ReadStringFromEnvVar("TORCH_MHLO_OP_WHITE_LIST", "");
+  std::ostringstream ostr;
+  ostr << "User defined white list: [";
+  std::istringstream f(custom_ops);
+  std::string s;
+  for (auto s : StrSplit(custom_ops, ';')) {
+    white_list.insert(std::string(s));
+    ostr << s << ", ";
+  }
+  ostr << "]";
+  LOG(INFO) << ostr.str();
 
-  static std::once_flag white;
-  std::call_once(white, []() {
-      auto custom_ops = env::ReadStringFromEnvVar("TORCH_MHLO_OP_WHITE_LIST", "");
-      std::ostringstream ostr;
-      ostr << "User defined white list: [";
-      std::istringstream f(custom_ops);
-      std::string s;
-      for (auto s : StrSplit(custom_ops, ';')) {
-        white_list.insert(std::string(s));
-        ostr << s << ", ";
-      }
-      ostr << "]";
-      LOG(INFO) << ostr.str();
-  });
   return white_list;
 }
 // clang-format off
 
+const std::unordered_set<std::string> &GetTorchMlirWhiteList() {
+  static std::unordered_set<std::string> white_list = CreateTorchMlirWhiteList();
```

Comment:
Nit: Could you please link the c++ 11 standard doc reference?

Reply:
please refer to this [SO answer](https://stackoverflow.com/a/1661564) or [C++11 standard](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2012/n3337.pdf) 6.7